getting_started/workflow/export/android_custom_build.rst

```diff
@@ -28,8 +28,9 @@ Go to the Project menu, and install the *Custom Build* template:
 Make sure export templates are downloaded. If not, this menu will help you
 download them.
 
-This will create an Gradle-based Android project in ``res://android/build`` and
-place a ``.gdignore`` file in ``res://android`` so the Godot filesystem ignores
+A Gradle-based Android project will be created under ``res://android/build``.
+
+Place a ``.gdignore`` file in ``res://android`` so the Godot filesystem ignores
```

Comment:
@eddex The original instruction is actually out of date and this instruction should not be needed as Godot already adds a `.gdignore` file in the `res://android/build` directory. 
You actually want the `res://android` directory to be visible so the user can install Android plugins if desired.

Reply:
alright, gonna change that :)